.travis.yml

```diff
@@ -26,19 +26,28 @@ before_script:
   - |
       if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       pip install git+https://github.com/SUPERAndroidAnalyzer/travis-cargo.git@fix-coveralls --user &&
-      export PATH=$HOME/.local/bin:$PATH
+      export PATH=$HOME/.local/bin:$PATH &&
+      (travis-cargo install rustfmt || true) &&
+      export PATH=$PATH:~/.cargo/bin &&
+      fi
+  - |
+      if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+      (cargo install rustfmt || true) &&
+      export PATH=$PATH:~/.cargo/bin
```

Comment:
Same as before, don't think we need to export the path. We probably don't want to run *rustfmt* on OS X builds. Ideally, I would only run it on `stable` Rust in Linux. That way builds don't get forever to work.